controls/cf_serverd.cf

```diff
@@ -32,9 +32,6 @@ body server control
       # Suggested value >= total number of hosts
       maxconnections        => "$(def.control_server_maxconnections)";
 
-      # Bind to all interfaces including ipv6
```

Comment:
@jimis @nickanderson I have tested with and without this patch on linux now. Even tested ipv6. Cannot see any difference, works fine with and without this patch.